docs/index.html

```diff
@@ -47,6 +49,8 @@ <h2 class="title"> Linux/Mac </h2>
             <h2 class="title"> Usage </h2>
             <p class="contents"> Run this command to list local and remote branches! </p>
             <pre> checkit </pre>
+            <img src="../assets/images/checkit-usage.gif" alt="checkit-log gif">

```

Comment:
@jma26 Take a look at the current readme and lets make sure we're not missing anything. 

There hsould be some text after the checkit usage code. + these two gifs are flipped.